### PR TITLE
Packaged content for MailRisk Solution

### PR DESCRIPTION
**Pull request by ingin97**
Required items, please complete
   
   Change(s):
   - Add solution package containing createUiDefinition.json and mainTemplate.json
   
   Reason for Change(s):
   - Make Solution ready for publishing.

   Testing Completed:
   - Have validated that the createUiDefinition.json and mainTemplate.json are passing validation. Though I have not been able to deploy the code in the Azure Function for the DataConnector automatically. Could need some help on this, if this is a requirement.

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hi @ingin97  looking into this

**Comment by ingin97** [comment]
Do you know how we can properly deploy the code when running the ARM template? If this is not a requirement at this stage: What is missing for this to get approved?

File: Solutions/MailRisk/Package/mainTemplate.json [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+        "APISecret": {
+            "type": "securestring",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The Secure Practice API key and secret are created in the <a href=\"https://manage.securepractice.co/settings/security\" target=\"_blank\">settings in the admin portal</a>."
+            }
+        }
+    },
+    "variables": {
+        "FunctionName": "[concat(toLower(parameters('FunctionName')), uniqueString(resourceGroup().id))]",
```
Comment (v-prasadboke) [inline Solutions/MailRisk/Package/mainTemplate.json:51]: Solution id not available 
please create the package using v2 tool

**Comment by v-prasadboke** [comment]
To repackage the solution 
Please go through this readme file
 [Azure-Sentinel](https://github.com/Azure/Azure-Sentinel)/[Tools](https://github.com/Azure/Azure-Sentinel/tree/master/Tools)/[Create-Azure-Sentinel-Solution](https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Create-Azure-Sentinel-Solution)/README.md

File: Solutions/MailRisk/Package/mainTemplate.json (hunk 1 above)
Comment (ingin97) [reply Solutions/MailRisk/Package/mainTemplate.json:51]: I manually added the mainTemplate.json last time as we do not have an arm template specific for the data connector, should I add that first, before repackaging with the tool?

**Comment by v-prasadboke** [comment]
Hi @ingin97 please insert the data file and meta data file which will be needed to create the package using the v2 tool.
Maintemplate can be deployed successfully in the workspace but due to missing metadata the solution is not visible in the content hub

**Comment by v-prasadboke** [comment]
Hello @ingin97 any updates

**Comment by v-prasadboke** [comment]
Hello @ingin97 any updates on the above

**Comment by v-atulyadav** [comment]
Hi @ingin97, please repackage the solution as per request. Thanks

**Comment by ingin97** [comment]
Hi @v-prasadboke Sorry for the late repsonse. I have submitted the files created by the script now.

**Comment by v-prasadboke** [comment]
Hello @ingin97 You will need to add input file as well,
If you don't mind should I add the input file in the solution.
For that will be needing additional data to create the input file

**Comment by ingin97** [comment]
Hi @v-prasadboke I added the input file as well now. Tell me if you need anything more.

**Comment by v-prasadboke** [comment]
Hello @ingin97 the input file which you have added should be in the 'Data' folder.
Please create a folder named as Data in the solution and move the input file in the folder.

**Comment by ingin97** [comment]
Hi @v-prasadboke The input file is now moved to the Data folder.

**Comment by v-prasadboke** [comment]
Hello @ingin97 there is still some issue while deploying the maintemplate

**Comment by ingin97** [comment]
Hi @v-prasadboke I am not sure what to do to fix it, as I used the script for creating the maintemplate. Do you have any ideas?

**Comment by ingin97** [comment]
@v-prasadboke Although it succeded when I deployed the template in the azure portal. Though the actual dataconnector code was not deployed, I think  we need help doing that.

**Comment by v-prasadboke** [comment]
Hello @ingin97 sorry for the late reply 
Let me see how i can help

**Comment by v-prasadboke** [comment]
Hello @ingin97 everything looks good to me

**Review by v-prasadboke — APPROVED** [review]

**Pull request merged**